app/queries/offers_to_chase_query.rb

```diff
@@ -0,0 +1,19 @@
+# Query to return AppliationChoices with offers whose offers have been issued
+# within an interval and whose candidates have not received emails chasing a
+# decision
+class OffersToChaseQuery
+  VALID_INTERVALS = [10, 20, 30, 40, 50].freeze
```

Comment:
`VALID_INTERVALS` seems underused here. Can't we use the keys in `DAYS_TO_CHASER_MAPPING` instead? `ArgumentError` could be potentially replaced with a `method_missing` on `ChaserSent`, but that's controversial.